backend/src/services/complectations/complectations.service.ts

```diff
@@ -0,0 +1,225 @@
+import { prisma } from '@data/prisma-client';
+
+import type {
+  ModelReturnedData,
+  ComplectationReturnedData,
+  OptionType,
+  ComplectationsInput,
+  ModelReturnedDbType,
+  ComplectationReturnedDbType,
+} from '@autoline/shared/common/types/types';
+
+const getComplectationsById = async (
+  input: ComplectationsInput,
+): Promise<ModelReturnedData | ComplectationReturnedData> => {
+  const { userId, modelId, complectationId } = input;
+  const data = (await prisma.complectation.findUnique({
+    where: {
+      id: complectationId,
+    },
+    select: {
+      users_wishlists: {
+        where: {
+          user_id: userId,
+        },
+        select: {
+          id: true,
+          complectation: {
+            select: {
+              id: true,
+            },
+          },
+        },
+      },
+      prices_ranges: {
+        select: {
+          price_start: true,
+          price_end: true,
+        },
+      },
+      color: { select: { name: true } },
+      engine_displacement: true,
+      engine_power: true,
+      drivetrain: { select: { name: true } },
+      fuel_type: { select: { name: true } },
+      transmission_type: { select: { name: true } },
+      options: {
+        select: {
+          option: { select: { name: true, type: true } },
+          important: true,
+        },
+      },
+    },
+  })) as ComplectationReturnedDbType;
```

Comment:
Why do we need explicit type coercion? Prisma already should generate a proper type